Release notes v1.29.0 (#2054)
agriculturegovau/agds-next: docs/content/updates/2025-06-03.mdx

---
date: 3rd June 2025
title: AgDS Beta v1.29.0 release
description: Fixes various hydration issues and simplifies use with Next.js’ App Router, improves clickable table rows, adds a new YouTube icon, plus various bug fixes and improvements.
---

## Updates

### [App layout](/components/app-layout)

- `AppLayoutSidebar` - Fix hydration mismatch warning when using Next.js’ App Router.

### [Avatar](/components/avatar)

- Fix live code examples when opened in Playroom.

### [Callout](/components/callout)

- Fix live code examples when opened in Playroom.

### [Cards](/components/cards)

- Fix live code examples when opened in Playroom.

### [Combobox](/components/combobox)

> - Prevent `useIsIos` from attempting to run on the server.

Reviewer: ```suggestion
- Prevent `useIsIos` from attempting to run in server side rendering.
```

Author: Hmm. Why do you think this is an improvement? I think mine is generic enough to cover that and server components. Or perhaps it should be inverted to "Only call `useIsIos` in browser environments"?

Author: Updated
